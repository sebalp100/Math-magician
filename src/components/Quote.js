import React from 'react';

const Quote = () => (
  <div className="homeContainer">
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean eu feugiat ex,
      et fermentum ipsum. Vestibulum ultrices dapibus enim egestas dictum. Suspendisse
      facilisis, purus et finibus tincidunt, nulla nulla eleifend mi, sit amet ullamcorper
      tellus ipsum in mi. Fusce imperdiet, augue nec iaculis congue, est mi accumsan elit,
      eget hendrerit risus leo a mauris. Donec id pretium eros, nec gravida metus.
    </p>
  </div>
);

export default Quote;
