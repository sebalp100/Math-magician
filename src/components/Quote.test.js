import * as React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

test('First Snapshot test', () => {
  const component = renderer
    .create(<Quote />)
    .toJSON();

  expect(component).toMatchSnapshot();
});
