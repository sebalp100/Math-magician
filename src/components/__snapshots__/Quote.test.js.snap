// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First Snapshot test 1`] = `
<div
  className="quoteContainer"
>
  <p>
    Pure mathematics is, in its way, the poetry of logical ideas.
    <br />
    Albert Einstein
  </p>
</div>
`;
